packages/babel-parser/test/fixtures/flow/type-annotations/typeof-reserved-invalid-2/input.js

```diff
@@ -0,0 +1,2 @@
+// @flow
+const x: typeof type.interface = "hi";
```

Comment:
I would report this as bug to the flow team, given that `type.interface` is valid in JS :thinking: